app/controlplane/pkg/conf/controlplane/config/v1/config.proto

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+package controlplane.config.v1;
```

Comment:
you probably need to place the buf.yaml to `conf` not o `conf/controlplane`

Reply:
Yes, that did the trick.